wbb/modules/images.py

```diff
@@ -34,23 +35,19 @@ async def dog(client, message):
 async def wall(client, message):
     app.set_parse_mode("markdown")
     m = await message.reply_text("Searching!")
-    try:
-        term = (message.text.split(None, 1)[1])
-    except IndexError:
-        await m.edit("`\"/wall\" needs a keyword argument`")
-
+    initial_term = (message.text.replace('/wall', ''))
+    term = initial_term.replace(' ', '%20')
     api = "https://wall.alphacoders.com/api2.0/get.php?auth=" + \
         "{}&method=search&term={}".format(WALL_API_KEY, term)
 
     json_rep = r.get(api).json()
     if not json_rep.get("success"):
-        await m.edit("FTS! something unusual happened, " +
-                     "report to support @Thepirategang")
+        await m.edit("FTS! something unusual happened, "
+                     + "report to support @Thepirategang")
     else:
         wallpapers = json_rep.get("wallpapers")
         if not wallpapers:
-            await m.edit(
-                "Found literally nothing!,You should work on your English.")
+            await m.edit("404")
```

Comment:
will this error message be deleted later? I recommend to use asyncio to delete the message some seconds later (such like deleting both the user command message and the "404" message after 5 seconds). also I think it's better to have original message instead of "404". maybe tell the user it's not found try to use other keyword.

Reply:
Yeah i wrote the correct message, and that message will be automatically deleted, check the last line of this file

Reply:
you should write a async function to delete the message some seconds later not delete it right after edit it.

something like this
https://github.com/konnovdev/cpop_bot/blob/dev/cpop_bot/text_handler.py#L138

Reply:
`await asyncio.sleep(10)`  how about this?

Reply:
maybe that's ok, but I think it's better to write it as a function.